denizen_scripts/HeroCraft/mobs/animal_mimic.dsc

```diff
@@ -0,0 +1,51 @@
+animal_mimic_events:
+    type: world
+    debug: false
+    events:
+        after cow spawns because NATURAL:
+            - stop if:<context.location.town.exists>
+            - define chance 0.5
+            - define chance:*:2 if:<context.location.world.has_storm>
+            - if <util.random_chance[<[chance]>]>:
+                - flag <context.entity> animal_mimic expire:5m
+        on entity_flagged:animal_mimic dies:
+            - stop if:<context.damager.is_player.not.if_null[true]>
+            - determine passively cancelled
+            - define location <context.entity.location>
+            - remove <context.entity> if:<context.entity.exists>
+            - spawn "<entity[animal_mimic].with[custom_name=<&f><context.entity.name.to_titlecase> Mimic].with_flag[mimic_drops:<context.drops>]>" <[location]> target:<context.damager.if_null[<[location].find_entities[player].within[50].first>]> persistent
+
+animal_mimic:
+    type: entity
+    debug: false
+    entity_type: skeleton
```

Comment:
Let's change this to something a bit more scary, like a creeper!

Reply:
Ok!

Reply:
I don't think a creeper is very scary, so I made it a husk.